src/app/hooks/useScopeParam.ts

```diff
@@ -14,14 +14,23 @@ type ScopeInfo = SearchScope & {
   valid: boolean
 }
 
+type Scope = {
+  layer: Layer | undefined
+  network: Network | undefined
```

Comment:
I don't think these fields can be undefined. It would be caught by loaders. I think useScopeParam only needs to check that one of the useRouteLoaderData is found

Reply:
But that was also true previously, so was `.valid` not needed, and always true?

Reply:
or do we never use the errors thrown in loaders

Reply:
large refactor in https://github.com/oasisprotocol/explorer/pull/1159

Reply:
ah I can see the plan is to merge this one first.